.dockerignore

```diff
@@ -1,4 +1,3 @@
 *
 !dev
-!packages/squidex
 !.github/scripts/squidex/requirements.txt
```

Comment:
can't we delete this `!.github/scripts/squidex/requirements.txt` as well? it seems this file does not exist